kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/QuestionsSettings.vue

```diff
@@ -36,7 +43,7 @@
           <KIconButton
             icon="plus"
             aria-hidden="true"
-            :disabled="questionCount >= maxQuestions || !questionCountIsEditable"
+            :disabled="questionCount >= inputMaxQuestions || !questionCountIsEditable"
```

Comment:
You can see in the diff that we were previously using `maxQuestions`. Let's continue to use that, rather than add another variable here.